### DISCUSSION TITLE
Keyball39 入力できないキーがあります。

### DISCUSSION THREAD

**Discussion by yukiWaka** [Yowkees/keyball#628] [Q&A] [answered]
他の方のQAも見て対応したのですが、解消しておらず、ご確認いただけないでしょうか
問題が２つあり、
① ダイオード、キーソケットの半田付けの見直しをしてもキー入力ができない
② ショートさせると入力できるが、キースイッチをはめると反応しなくなる

① 
　別のQAでもあったように、キーソケットが浮いていないか確認、半田を足したり、
　ダイオードの見直しも行ったのですが、解消していません。
　問題のキースイッチが赤マル。青丸のダイオードの見直しは行いました。
　他に関係する部分があれば、教えていただけないでしょうか？
<img width="695" alt="スクリーンショット 2024-07-17 12 39 23" src="https://github.com/user-attachments/assets/62b6622c-48d5-4d1b-8ce0-a2921dfea48d">

②
　キーソスイッチのはまっていない箇所が、ピンセットでショートさせるとキー入力できているのですが、
　キーソスイッチをはめると反応しなくなってしまいます。
　他の正常に動作しているキースイッチをはめても、変わらずだったのでキースイッチ側の問題では無さそうです。

　キーソケットを交換してみるのが良いでしょうか？
　ご助言いただけないでしょうか。
<img width="593" alt="スクリーンショット 2024-07-17 12 42 35" src="https://github.com/user-attachments/assets/ea42fb38-6f88-47fa-96dc-ff09d992ef4b">

**Comment by yukiWaka** [comment]
②に関しては↓のQAを見て、改めて半田を足したところ解消しました。
https://github.com/Yowkees/keyball/discussions/615

**Reply by yukiWaka** [reply, marked as answer]
①についても自己解決しました。
ダイオードを半田付けしなおしたところ解消しました。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。ご自身で解決していただき、誠にありがとうございます。

ぜひ末永くご愛用いただければ幸いです。
